package.json

```diff
@@ -149,6 +149,24 @@
           "default": [],
           "description": "Per-workspace list of module directories for the language server to read"
         },
+        "terraform-ls.terraformExecPath": {
+          "scope": "resource",
+          "type": "string",
+          "default": [],
+          "description": "Path to Terraform binary"
+        },
+        "terraform-ls.terraformExecTimeout": {
+          "scope": "resource",
+          "type": "string",
+          "default": [],
+          "description": "Overrides Terraform execution timeout (e.g. 30s)"
+        },
+        "terraform-ls.terraformExecLogFilePath": {
+          "scope": "resource",
+          "type": "string",
+          "default": [],
```

Comment:
The default type seems wrong (array) for string arguments but more importantly I don't think we need any defaults for these options in particular? Do you mind removing the defaults?

Reply:
You're right, didn't notice that while copying over this block from an existing piece. Updated.